packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2798,6 +2798,10 @@ class InputDecoration {
   /// of [InputDecoration] that changes the color of the label to the
   /// [InputDecoration.errorStyle] color or [ColorScheme.error].
   ///
+  /// When [InputDecoration.label] is shown above the input field, it is scaled down
+  /// to 0.75. So, any given font size needs to be changed
+  /// accordingly that it will be scaling down at the end.
```

Comment:
```suggestion
  /// When the input field receives focus, the font size of [InputDecoration.label] is 
  /// scaled down by 75%.
```

Reply:
What do you think @rkishan516?

Reply:
@Piinks We only use `floatingLabelStyle` when field has focus. I don't know, we should mention that or not. Also, we should clearly tell user that it's not font size changing its label itself scaling down by 75%.
```dart
/// The [InputDecoration.label] is scaled down to 75% when input field receives focus.
/// Adjust the font size accordingly to ensure it appears as intended after scaling.
```

Reply:
May be we don't need 
```dart
/// Adjust the font size accordingly to ensure it appears as intended after scaling.
```

But to be on safe side, I want user to know that he should change `fontSize` knowing this fact.

Reply:
They may not need to change the font size if the default behavior is what they want. This seems like for of a 'for your information' doc, rather than a call to action. If it were a bug, we would just fix the font scaling for them. :)

Reply:
Yes @Piinks, I think we need to go back to https://github.com/flutter/flutter/issues/151406, to understand the issue. That says let suppose if we are giving x font size for floatingLabelStyle, but actually its visible size is not x, reason the whole label is scaled down by 75%.

Reply:
Ok! From the original issue:

> Nothing in the [official docs](https://api.flutter.dev/flutter/material/InputDecoration/floatingLabelStyle.html) even mentions that this would happen.

I think it is fine to just mention the behavior, since it is working as expected. Users do not necessarily need to make any changes, which is why I suggested removing the recommendation.

Reply:
Sure, I have done the change.